ert-tests/test_files_secondary/.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
```

Comment:
Is there a better place this can go? I don't want it to go in the submodule, it seems specific to this plugin.